website/content/docs/k8s/crds/index.mdx

```diff
@@ -180,306 +173,3 @@ to be deleted since that would put Consul into a broken state.
 
 In order to delete the `ServiceDefaults` config, you would need to first delete
 the `ServiceSplitter`.
-
-## Specifications
```

Comment:
These specs are removed since they're documented in the main config entry docs now.